ignek_intranet/modules/common/src/main/java/com/ignek/intranet/common/service/EmployeeService.java

```diff
@@ -0,0 +1,35 @@
+package com.ignek.intranet.common.service;
+
+import java.io.ByteArrayOutputStream;
+import java.util.List;
+
+import com.ignek.intranet.common.employee.response.EmployeeResponse;
+import com.ignek.intranet.employee.model.Employee;
+import com.itextpdf.text.DocumentException;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.search.Hits;
+import com.liferay.portal.kernel.search.ParseException;
+import com.liferay.portal.kernel.search.SearchException;
+
+public interface EmployeeService {
+
+	EmployeeResponse addUser(long empId, long userId, long companyId, String firstName, String lastName,
```

Comment:
We need to make it common method to save employee

New method name : saveEmployee